DER/DerivativesContracts/Options.rdf

```diff
@@ -246,15 +213,15 @@
 		<fibo-fnd-utl-av:explanatoryNote xml:lang="en">If the option is exercised, delivery is in the form of the corresponding interest rate future.</fibo-fnd-utl-av:explanatoryNote>
 	</owl:Class>
 	
-	<owl:Class rdf:about="&fibo-der-der-opt;IntrinsicValue">
+	<owl:Class rdf:about="&fibo-der-drc-opt;IntrinsicValue">
 		<rdfs:subClassOf rdf:resource="&fibo-fnd-acc-cur;MonetaryMeasure"/>
 		<rdfs:label xml:lang="en">intrinsic value</rdfs:label>
 		<skos:definition xml:lang="en">measure of what an asset is worth, i.e. with respect to its current price</skos:definition>
```

Comment:
@rivettp We left it general in case someone wanted to use it in a broader sense.  I seem to recall there was discussion about it in the FCT, but don't remember whether or not you were on that call.